src/libs/Navigation/AppNavigator/AuthScreens.tsx

```diff
@@ -238,6 +179,42 @@ function AuthScreens({session, lastOpenedPublicRoomID, initialLastUpdateIDApplie
         canBeMissing: true,
     });
     const [onboardingCompanySize] = useOnyx(ONYXKEYS.ONBOARDING_COMPANY_SIZE, {canBeMissing: true});
+    const [personalDetailsList] = useOnyx(ONYXKEYS.PERSONAL_DETAILS_LIST, {canBeMissing: true});
```

Comment:
Can we add a selector here? `personalDetailsList` is an object that can contain thousands of records, let's select just the current user and ideally the timezone only, since that's all that's used here